Makefile

```diff
@@ -1,7 +1,8 @@
 DEVENV = devenv
 NODE_MODULES = node_modules
 PROJECT = js-libjuju
-SYSDEPS = build-essential python3-virtualenv tox
+SCHEMA = schemas/schema.json
```

Comment:
Can we have this save the schema with a version number of some sort? Even if it's the juju 6char sha?

Reply:
Yes, but it would complicate things a little bit. Maybe I could save a separate file with metadata about the schema I guess.

Reply:
Yes ok I'll do that in a follow-up.